telemetry/qml/TelemetryPermissionDialog.qml

```diff
@@ -57,7 +57,7 @@ Rectangle {
             color: "#00447a"
             font.pixelSize: 28
 
-            text: qsTr("Help us improve MuseScore")
+            text: qsTr("Help us to improve MuseScore")
```

Comment:
Still, I'm not sure about this.

Reply:
https://english.stackexchange.com/questions/3578/what-is-the-correct-way-to-use-infinitive-after-the-verb-help-with-or-without might be helpful.

Reply:
Or `Help us improving MuseScore`?
Native speakers wanted ;-)

Reply:
@Jojo-Schmitz there is no way of using "Help us improving": https://dictionary.cambridge.org/grammar/british-grammar/help-somebody-to-do.

I'm convinced that "Help us improve" is the correct form for native speakers. Others will use translation or just won't care :) Finally, we use the same wording other companies do: https://ec.europa.eu/solvit/help-us-improve/index_en.htm, https://helpx.adobe.com/x-productkb/multi/disable-help-improve-adobes-products.html

Reply:
@Jojo-Schmitz could you please revert that change?

Reply:
Ok, but that has to wait till at leat tomorrow afternoon, if you need it earlier you could cherry-pick this, commit and leave out this particular chunk maybe?

Reply:
Ah, I see you just did ;-)